cmd/shadowsocks-local/guess.go

```diff
@@ -0,0 +1,138 @@
+package main
```

Comment:
this file should be in `shadowsocks` package but not the client `main` package

Reply:
Done.